src/platform/plugins/shared/discover/public/context_awareness/profile_providers/common/patterns/pattern_cell_renderer.tsx

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React, { useMemo } from 'react';
+import type { FC } from 'react';
+import { EuiSpacer, EuiText, useEuiTheme } from '@elastic/eui';
+import type { DataTableRecord } from '@kbn/discover-utils';
+import { extractKeywordsFromRegex } from '@kbn/aiops-log-pattern-analysis';
+
+export interface Props {
```

Comment:
Doesn't seem like this Props interface needs to be exported. But if we are exporting the type so others can consume, potentially rename it to PatternCellRendererProps for clarity?

Reply:
Updated in https://github.com/elastic/kibana/pull/222871/commits/3e7f150ea904d93772d8575f5f6d6b163ec3bbd9